app/src/main/java/org/koitharu/kotatsu/base/ui/widgets/ChipsView.kt

```diff
@@ -4,17 +4,18 @@ import android.content.Context
 import android.util.AttributeSet
 import android.view.View.OnClickListener
 import androidx.annotation.DrawableRes
+import androidx.core.content.ContextCompat
 import androidx.core.view.children
-import com.google.android.material.R
 import com.google.android.material.chip.Chip
+import com.google.android.material.chip.ChipDrawable
 import com.google.android.material.chip.ChipGroup
+import org.koitharu.kotatsu.R
 import org.koitharu.kotatsu.utils.ext.getThemeColor
 
 class ChipsView @JvmOverloads constructor(
 	context: Context,
-	attrs: AttributeSet? = null,
-	defStyleAttr: Int = R.attr.chipGroupStyle
```

Comment:
Why you remove this?

Reply:
Oh, I tried to make the attributes of the chip group styling, I temporarily removed it, but I forgot to revert it